test/controllers/products_controller_test.rb

```diff
@@ -0,0 +1,33 @@
+require "test_helper"
+
+class ProductsControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
```

Comment:
As we are not dealing with rspec or test case now so please don't push test files to the repository